From 4290835aab8b6fc4cccbd2d0da84b806237bf0b4 Mon Sep 17 00:00:00 2001
From: Liu Ying <b17645@freescale.com>
Date: Tue, 28 Oct 2008 20:24:24 +0800
Subject: [PATCH] ENGR00094435 MX51: Fix v4l2 still image capture issue

When still image capture stops, add the reset of the waiting
number of first several frames. This makes still image feature
work continuously.

Signed-off-by: Liu Ying <b17645@freescale.com>
---
 drivers/media/video/mxc/capture/ipu_still.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ipu_still.c b/drivers/media/video/mxc/capture/ipu_still.c
index 746df1e..d82a5e6 100644
--- a/drivers/media/video/mxc/capture/ipu_still.c
+++ b/drivers/media/video/mxc/capture/ipu_still.c
@@ -165,6 +165,8 @@ static int prp_still_stop(void *private)
 	cam_data *cam = (cam_data *) private;
 	int err = 0;
 
+	callback_eof_flag = 0;
+
 #ifdef CONFIG_MXC_IPU_V1
 	ipu_free_irq(IPU_IRQ_SENSOR_EOF, NULL);
 	ipu_free_irq(IPU_IRQ_SENSOR_OUT_EOF, cam);
-- 
1.5.4.4

